packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -176,7 +177,8 @@ Enumerates Groups in your organization with pagination. A subset of Groups can b
 | after     | Specifies the pagination cursor for the next page of Groups                                | Query     | String   | FALSE    |         |
 | filter    | [Filter expression](/docs/reference/api-overview/#filtering) for Groups      | Query     | String   | FALSE    |         |
 | limit     | Specifies the number of Group results in a page                                            | Query     | Number   | FALSE    | 10000   |
-| q         | Searches the `name` property of Groups for matching value                                  | Query     | String   | FALSE    |         |
+| q         | Find groups that matches the `name` property                                               | Query     | String   | FALSE    |         |
```

Comment:
Nit: singular/plural agreement.  It's `groups that match` or `group that matches`.